templates/etc/supervisor/supervisord.conf.j2

```diff
@@ -14,6 +14,7 @@ password = {{ supervisor_inet_http_server_password }}
 {% endif %}
 
 [supervisord]
+nodaemon=true
```

Comment:
How about adding a variable, like the rest of supervisor settings, so people can enable/disable that without editing the role itself ?